share/github-backup-utils/ghe-gc-disable

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#/ Usage: ghe-gc-disable [<option>...] <host>
+#/ Helper to disable and drain GC operations on a GitHub Enterprise server.
```

Comment:
Even if it's for internal use, we may want to explain the options here:

```
#/ Usage: Usage: ghe-gc-disable [<option>...] <host>
#/
#/ Helper to disable and drain GC operations on a GitHub Enterprise server.
#/
#/ OPTIONS:
#/   -F <required_arg>                Blah blah
```

We still need to add a styleguide, but that'll be the preferred format probably.